lib/airbrake/rails/app.rb

```diff
@@ -45,7 +45,7 @@ def self.recognize_route(request)
             # versions support.
             #
             # https://github.com/rails/rails/commit/5460591f0226a9d248b7b4f89186bd5553e7768f
-            next if route.respond_to?(:glob?) && route.glob?
+            next if route.nil? && route.respond_to?(:glob?) && route.glob?
```

Comment:
I think we can safely remove `route.respond_to?(:glob?)` (and the todo comment above) since we no longer test against Rails 4.1 and lower.

Can `route` actually be ever `nil`? In the failing scenario it's not `nil`. What is `nil` is `route.path.ast`. Should we be checking for `route.path.ast.nil?` since that's the implementation of `glob?` and the call that fails?

Reply:
https://github.com/airbrake/airbrake/pull/1206#discussion_r782924573